contracts/PoolManager.sol

```diff
@@ -108,6 +108,25 @@ contract PoolManager is IPoolManager, Fees, NoDelegateCall, ERC1155, IERC1155Rec
         return pools[id].positions.get(owner, tickLower, tickUpper);
     }
 
+    function _accountNoOp(PoolKey memory key, BalanceDelta hookDelta) internal {
+        // NOTE: we are in the lock state of the user, not the hook, unless the hook is the user
```

Comment:
I think it'd be neater to take comments out of the function, they feel a bit inconsistent, and the code feels self-exlanatory, and just write one comment above along the lines of:
```
/// @dev on noOp, rebalance 1155 balances of hook and account deltas to the locker
```